mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -138,6 +139,11 @@ func Provider() *schema.Provider {
 			    Optional: true,
 			},
 
+			"request_reason": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Can we defined a `DefaultFunc` like in `user_project_override` above? That'll put the environment variable logic here too.